app/jobs/resend_consultee_emails_job.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+class ResendConsulteeEmailsJob < NotifyEmailJob
```

Comment:
This looks very similar to `SendConsulteeEmailsJob`. Am I right in thinking that the only differences are the `message` and `date` parameters and the logic around them? I wonder if it's worth just adding these as optional parameters to the existing job?